scripts/azureml-assets/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ### 🐛 Bugs Fixed
 
+## 1.16.13 (2023-10-12)

```

Comment:
Just remember set the date when this actually is released.

Reply:
Copy paste fail. Updating to today hoping we can get this PR approved soon.